lock-impl/src/test/java/com/palantir/lock/impl/LockServiceImplTest.java

```diff
@@ -48,7 +49,7 @@
 
 public final class LockServiceImplTest {
     public static final long SLOW_LOG_TRIGGER_MILLIS = LockServiceImpl.DEBUG_SLOW_LOG_TRIGGER_MILLIS + 10;
-    private static final String TEST_LOCKID = "test_lockId";
+    private static final LockDescriptor TEST_LOCK = StringLockDescriptor.of("test_lockId");
```

Comment:
I'd just create a new constant called TEST_LOCKID and construct the TEST_LOCK from that, to save the unnecessary change that's below for each log line

Reply:
turns out we need to touch every line anyways due to the additional log argument